CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 
 CMAKE_MINIMUM_REQUIRED(VERSION 3.5.1)
 
-project(libgit2 C)
+project(libgit2 VERSION "1.0.0" LANGUAGES C)
```

Comment:
1.0.1 now.  Sorry it took so long to review.

Reply:
Indeed. Rebased the MR and fixed the version, thanks!